flutter/pubspec.yaml

```diff
@@ -19,6 +19,7 @@ dependencies:
   package_info_plus: '>=1.0.0 <=5.0.1'
   meta: ^1.3.0
   ffi: ^2.0.0
+  connectivity_plus: ^5.0.2
```

Comment:
Pls check if we can do this without having to add another package, only if the complexity is reasonable